launch/tier4_control_launch/launch/control.launch.xml

```diff
@@ -0,0 +1,186 @@
+<launch>
```

Comment:
Will you keep both `.xml` and `.py` or remove `.py` later?
Anyway, I recommend creating a PR to delete `.py` because that enables you to track changes in `.py` easily.

Reply:
Also, it helps reviewers check the diff.

Reply:
Regarding the timing of `.py` deletion, it would be good to have several transition periods because it breaks the interface between `autoware_launch`.

Reply:
! will remove python launch later because we also need to change [here](https://github.com/autowarefoundation/autoware_launch/blob/c6a3df8be00302a72f6fd2c447db7faf0d80475f/autoware_launch/launch/components/tier4_control_component.launch.xml#L11)

Reply:
Cool!

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2834